### ISSUE TITLE
There are results for an image, but there are no results there

### ISSUE THREAD

**Issue by little-wow** [creativecommons/cccatalog-frontend#147]
<img width="1439" alt="screen shot 2019-01-16 at 10 30 59 am" src="https://user-images.githubusercontent.com/7093838/51268331-dff57780-198d-11e9-9cb0-e1b1223fcc8a.png">

Bummer!

**Comment by aldenstpage** [comment]
Mismatches in result count and actual results are caused by broken link filtering. The app checks that every image exists before displaying the search results. These images were either deleted, moved, or did not load quickly enough. I'll take a look at the reason these particular images are broken (maybe we need to give them more time to load before filtering them). I'll also change things so the search result count is 0 if all of the images have been filtered out.

**Comment by janeatcc** [comment]
This is also happening with a search for images on Thingiverse: https://ccsearch-dev.creativecommons.org/search?q=horse&provider=thingiverse&li=&lt=

**Comment by aldenstpage** [comment]
Hi @janeatcc, @brenoferreira, I took a quick look at the Thingiverse example and it looks like an error on the frontend is preventing the search response from rendering. Looking in the browser console, the API appears to return some images, see [here](http://api.creativecommons.engineering/image/search?q=horse&provider=thingiverse&li=&lt=).

This is what I see in the network console:
```
vue.esm.js:1741 Error: Cannot find module './'.
    at L (.*$:104)
    at e (.*$:99)
    at a.getProviderLogo (SearchGridCell.vue:64)
    at a.render (SearchGridCell.vue?ad6f:1)
    at a.t._render (vue.esm.js:4544)
    at a.<anonymous> (vue.esm.js:2788)
    at Le.get (vue.esm.js:3142)
    at new Le (vue.esm.js:3131)
    at vue.esm.js:2795
    at a.pn.$mount (vue.esm.js:8540)
    at a.pn.$mount (vue.esm.js:10939)
    at init (vue.esm.js:4137)
    at vue.esm.js:5608
    at f (vue.esm.js:5555)
    at b (vue.esm.js:5735)
    at vue.esm.js:5862
    at S (vue.esm.js:5936)
    at vue.esm.js:5822
    at S (vue.esm.js:5936)
    at a.__patch__ (vue.esm.js:6096)
    at a.t._update (vue.esm.js:2670)
    at a.<anonymous> (vue.esm.js:2788)
    at Le.get (vue.esm.js:3142)
    at Le.run (vue.esm.js:3219)
    at Te (vue.esm.js:2981)
    at Array.<anonymous> (vue.esm.js:1837)
    at Wt (vue.esm.js:1758)
```

This is a separate problem from the one that caused @little-wow's issue; that one is definitely from link filtering, but I haven't had a chance to check the status of the content yet.

I didn't see any problems with the Rijksmuseum example.

**Comment by brenoferreira** [comment]
@aldenstpage this Thingiverse bug you mention seems to be a different problem. I'll try to find what's causing this and will file a separate issue. Thanks for spotting this :)

**Comment by janeatcc** [comment]
same thing happening for thingiverse: https://ccsearch.creativecommons.org/search?q=cat&provider=thingiverse&li=&lt=

**Comment by janeatcc** [comment]
same for WORMS: https://ccsearch.creativecommons.org/search?q=fish&provider=WoRMS&li=&lt=

keeps happening for all providers really?

**Comment by aldenstpage** [comment]
Fixed in cccatalog-api v0.20.3; the count is zero.
